.env-devel

```diff
@@ -108,7 +108,7 @@ DYNAMIC_SIDECAR_API_SAVE_RESTORE_STATE_TIMEOUT=3600
 # DIRECTOR_V2 ----
 DYNAMIC_SCHEDULER_LOGLEVEL=DEBUG
 DYNAMIC_SCHEDULER_PROFILING=1
-DYNAMIC_SCHEDULER_STOP_SERVICE_TIMEOUT=3600
+DYNAMIC_SCHEDULER_STOP_SERVICE_TIMEOUT=PT1H
```

Comment:
for info, I use "01:00:00" instead of PT1H... I find it more readable...

Reply:
yes, please